src/app/modules/item/http-services/create-item.service.ts

```diff
@@ -27,8 +25,6 @@ export class CreateItemService {
   }
 
   create(newItem: NewItem): Observable<string> {
```

Comment:
I didn't noticed it yesterday, but using an interface it and passing it to the `post` is not correct. Your interface just mean that there are at least these attributes.. but actually the object may have other attributes and they will all be sent the the backend, which is not ok.
Probably you could keep an interface with camelcase attribute, and create the arg of post directly at the call:
```
.post<ActionResponse<NewItemData>>(`${appConfig().apiUrl}/items`, {
  title: input.title, 
  ... 
  language_tag: input.languageTag,
  ...
})
```

Reply:
Done